x-pack/plugins/synthetics/server/synthetics_service/formatters/tcp.ts

```diff
@@ -5,21 +5,15 @@
  * 2.0.
  */
 
+import { tcpFormatters as baicTCPFormatters } from '../../../common/formatters/tcp/formatters';
 import { Formatter, commonFormatters } from './common';
 import { tlsFormatters } from './tls';
 import { ConfigKey, TCPFields } from '../../../common/runtime_types/monitor_management';
 
 export type TCPFormatMap = Record<keyof TCPFields, Formatter>;
-
 export const tcpFormatters: TCPFormatMap = {
+  ...baicTCPFormatters,
```

Comment:
Same here, feel like TCP should come last, but I see you didn't change anything there so I don't know. Maybe there's some context I don't remember.